src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/DenoisedCopyRatioCollection.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.LinearNonLocatableCopyRatio;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.GermlineCNVNamingConstants;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+
+import java.io.File;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Collection of denoised copy ratios generated by {@link GermlineCNVCaller}
+ *
+ * @author Andrey Smirnov &lt;asmirnov@broadinstitute.org&gt;
+ */
+public class DenoisedCopyRatioCollection extends AbstractSampleRecordCollection<LinearNonLocatableCopyRatio> {
+
+    public DenoisedCopyRatioCollection(final File inputFile) {
+        super(inputFile,
```

Comment:
Nitpicking, but I would put `inputFile` on its own line to get cleaner indentation.  I see other calls to `super` in the collection classes are inconsistent, though, so feel free to disregard... :(

Reply:
Hmm there is a big space if you put it on its own line, looks a little strange..